src/modules/RA/DataLayer/views/List.js

```diff
@@ -30,6 +32,29 @@ const ListFilters = props => (
     >
       <SelectInput />
     </ReferenceInput>
+    <ReferenceInput
+      source="group__view"
+      reference={RES_VIEW}
+      label="datalayer.form.view"
+    >
+      <SelectInput />
+    </ReferenceInput>
+    <BooleanInput
+      source="active_by_default"
+      label="resources.datalayer.fields.active_by_default_pastpart"
+    />
+    <BooleanInput
+      source="table_enable"
+      label="resources.datalayer.fields.table_enable"
```

Comment:
react-admin provide a default label corresponding to the form element path _(which in this case is `resources.datalayer.fields.table_enable`)_. So you don't need to add manuel label if you do not require a different translation key.

```suggestion
```